rtf_vis_tool/src/Components/DivNode.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import '.././stylesheets/Div_Node.css';
+
+//Basic Div Component repeatedly rendered in DivGraph.js
```

Comment:
was DivGraph renamed?

Reply:
yup. forgot to update the comment. I've updated it now.